lib/request-base.js

```diff
@@ -130,6 +130,50 @@ RequestBase.prototype.timeout = function timeout(options){
 };
 
 /**
+ * Set number of retry attempts on error.
+ *
+ * Failed requests will be retried 'count' times if timeout or err.code >= 500.
+ *
+ * @param {Number} count
+ * @return {Request} for chaining
+ * @api public
+ */
+
+RequestBase.prototype.retry = function retry(count){
+  this._maxRetries = count || 1;
+  this._retries = 0;
```

Comment:
Sorry, I've meant it's set in https://github.com/visionmedia/superagent/pull/1155/files#diff-c24ce7e3da4c0e4ff811a2b6a76f8bd9R145

It's not an error, but it's redundant. `_retries` are never increased before `.retry()` is called, so there's no point zeroing something that's zero anyway.